### ISSUE TITLE
Nereond created files should be monitored

### ISSUE THREAD

**Issue by ronaldtse** [riboseinc/nereond#1]
To ensure they’re not being changed, such as to prevent configuration drift. If changed it should generate an event (Nerelog event). 

On Linux we can use inotify and on macOS fswatch. When possible, the named pipe can also be used.

**Comment by drystone** [comment]
Right. I've been wondering about inotify but was concerned about it's portability.

Is this so Nereond can detect configuration modification via an external mechanism?

Inotify would be a great way for Riffol to pick up configuration changes and trigger service restarts. 

Would there be any need for the named pipe in this case? Or is there other information for Nereond->Riffol to communicate?
